HotorNot/app/src/main/java/com/example/hotornot/MainActivity.kt

```diff
@@ -0,0 +1,26 @@
+package com.example.hotornot
+
+import androidx.appcompat.app.AppCompatActivity
+import android.os.Bundle
+import androidx.databinding.DataBindingUtil
+import androidx.navigation.NavController
+import androidx.navigation.fragment.NavHostFragment
+import androidx.navigation.fragment.findNavController
+import androidx.navigation.ui.setupActionBarWithNavController
+import com.example.hotornot.databinding.ActivityMainBinding
+
+class MainActivity : AppCompatActivity() {
+
+    private lateinit var binding: ActivityMainBinding
+    private lateinit var navController: NavController
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        binding = DataBindingUtil.setContentView(this,R.layout.activity_main)
+
+        val navHostFragment =
+            supportFragmentManager
+                .findFragmentById(R.id.navHostFragment) as NavHostFragment
+        navController = navHostFragment.findNavController()
```

Comment:
I'm not sure,but are you hidden action bar ?if you don't hide it, hide it with supportActionBar?.hide() or or something like this.